assets/js/src/editor/sidebar/edit.js

```diff
@@ -119,16 +119,21 @@ function Edit( { media, headerImageId, onUpdateImage, onRemoveImage } ) {
 }
 
 export default compose(
-	// Render only for posts.
+	// Render only for supported post types.
 	withSelect( ( select ) => {
+		const { getEditedPostAttribute } = select( 'core/editor' );
+		const { getPostType } = select( 'core' );
 		return {
-			isPostTypePost: includes(
-				['post', 'page'],
-				select( 'core/editor' ).getEditedPostAttribute( 'type' )
-			),
+			postType: getPostType( getEditedPostAttribute( 'type' ) ),
 		};
 	} ),
-	ifCondition( ( { isPostTypePost } ) => isPostTypePost ),
+	ifCondition( ( { postType } ) => {
+		let isSupported = false;
+		if ( postType ) {
+			isSupported = postType.supports[ 'post-header-image' ];
```

Comment:
`isSupported` will be `undefined` or `null` if support is not set? Should make sure that it's always a boolean.

Reply:
How would you do that?

Reply:
Blauäugig wie ich bin, schlag ich mal sowas vor: 😂🙈
```suggestion
			isSupported = !! postType.supports[ 'post-header-image' ];
```